tests/acceptance/features/bootstrap/SpacesContext.php

```diff
@@ -874,4 +874,105 @@ public function sendCreateFolderRequest(
 
 		return HttpRequestHelper::sendRequest($fullUrl, $xRequestId, $method, $user, $password, $headers);
 	}
+
+	/**
+	 * @When /^user "([^"]*)" changes the name to "([^"]*)" of the "([^"]*)" space$/
+	 *
+	 * @param string $user
+	 * @param string $newName
+	 * @param string $spaceName
+	 *
+	 * @return void
+	 * @throws GuzzleException
+	 * @throws Exception
+	 */
+	public function updateSpaceName(
+		string $user,
+		string $newName,
+		string $spaceName
+	): void {
+		$space = $this->getSpaceByName($spaceName);
+		$spaceId = $space["id"];
+
+		$bodyData = ["Name" => $newName];
+		$body = json_encode($bodyData, JSON_THROW_ON_ERROR);
+
+		$this->featureContext->setResponse(
+			$this->sendUpdateSpaceRequest(
+				$this->featureContext->getBaseUrl(),
+				$user,
+				$this->featureContext->getPasswordForUser($user),
+				$body,
+				$spaceId
+			)
+		);
+	}
+
+	/**
+	 * @When /^user "([^"]*)" changes the quota to "([^"]*)" of the "([^"]*)" space$/
+	 *
+	 * @param string $user
+	 * @param int $newQuota
+	 * @param string $spaceName
+	 *
+	 * @return void
+	 * @throws GuzzleException
+	 * @throws Exception
+	 */
+	public function updateSpaceQuota(
+		string $user,
+		int $newQuota,
+		string $spaceName
+	): void {
+		$space = $this->getSpaceByName($spaceName);
+		$spaceId = $space["id"];
+
+		$bodyData = ["quota" => ["total" => $newQuota]];
+		$body = json_encode($bodyData, JSON_THROW_ON_ERROR);
+
+		$this->featureContext->setResponse(
+			$this->sendUpdateSpaceRequest(
+				$this->featureContext->getBaseUrl(),
+				$user,
+				$this->featureContext->getPasswordForUser($user),
+				$body,
+				$spaceId
+			)
+		);
+	}
+
+	/**
+	 * Send Graph Update Space Request
+	 *
+	 * @param  string $baseUrl
+	 * @param  string $user
+	 * @param  string $password
+	 * @param  mixed $body
+	 * @param  string $spaceId
+	 * @param  string $xRequestId
+	 * @param  array  $headers
+	 *
+	 * @return ResponseInterface
+	 *
+	 * @throws GuzzleException
+	 */
+	public function sendUpdateSpaceRequest(
+		string $baseUrl,
+		string $user,
+		string $password,
+		$body,
+		string $spaceId,
+		string $xRequestId = '',
+		array $headers = []
+	): ResponseInterface {
+		$fullUrl = $baseUrl;
+		if (!str_ends_with($fullUrl, '/')) {
+			$fullUrl .= '/';
+		}
+		$fullUrl .= "graph/v1.0/Drive($spaceId)";
+		$method = 'PATCH';
+
+		return HttpRequestHelper::sendRequest($fullUrl, $xRequestId, $method, $user, $password, $headers, $body);
+	}
+
```

Comment:
extra line
`make test-php-style` complains
you can run `make test-php-style-fix` to fix it automatically

Reply:
I deleted it, I don't want to create a bug on that is still " in progress"